pkg/cache/cache.go

```diff
@@ -38,6 +38,9 @@ const (
 	// SSMGetParametersByPathTTL is the time to drop SSM Parameters by path data. This only queries EKS Optimized AMI
 	// releases, so we should expect this to be updated relatively infrequently.
 	SSMGetParametersByPathTTL = 24 * time.Hour
+	// VMMemoryOverheadMebibytesTTL is the time to drop vm memory overhead data per-instance type
+	// if it is not updated by a node creation event or refreshed during controller reconciliation
+	VMMemoryOverheadMebibytesTTL = 36 * time.Hour
```

Comment:
How would you suggest offloading cached values that are no longer needed? I think having some form of expiration here would be good, would extending this to 30d be sufficient?

Reply:
If the AMIs have changed, the key should change as well right? I suppose we should have an expiration period to prevent this cache from growing indefinitely, but it could definitely be longer.

Reply:
Yep exactly, growing indefinitely is what I was trying to avoid with this. Happy to adjust to whatever's reasonable and will start with 30d for now.

Reply:
Configured for 60d